src/include/migraphx/check_context.hpp

```diff
@@ -36,7 +38,22 @@ struct check_context
 {
     struct op : auto_register_op<op>
     {
-        std::string name() const { return "check_context::" + get_type_name<T>(); }
+        static std::string name()
```

Comment:
This should still be a method. What I meant was to have a static function to compute the type and then assign it to a local static variable so its only computed once:

```cpp
static std::string compute_name()
{
    ...
}
std::string name() const
{
    static auto s = compute_name();
    return s;
}
```

Reply:
Done.